pkg/gadgets/filetop/tracer/bpf/filetop.bpf.c

```diff
@@ -0,0 +1,108 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
I like this notation. It makes it immediate that the file contains bpf code.

Reply:
I am OK with any notation and I understand your argument.
But we need to have the same for all our gadgets to bring some harmony to our code base.

For the moment, none of our gadget use the following notation:

```bash
$ find . -name '*.bpf.c'
$
```

You can merge it like this (I already gave my LGTM) but in this case it would be cool to decide we would need to rename the others.